draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -173,7 +173,27 @@ of hardware, software, entitlement, and inventory topology mapping.
 
 The YANG data model defined in this document conforms to the Network Management Datastore Architecture {{!RFC8342}}.
 
-## Terminology and Notations
+## Editorial Note (To be removed by RFC Editor)
+
+  > Note to the RFC Editor: This section is to be removed prior to publication.
+
+   This document contains placeholder values that need to be replaced
+   with finalized values at the time of publication.  This note
+   summarizes all of the substitutions that are needed.
+
+   Please apply the following replacements:
+
+   *  XXXX --> the assigned RFC number for this I-D
+
+   *  2024-03-04 --> the actual date of the publication of this document
```

Comment:
```suggestion
```

This date will keep changing every time we update the draft YANG model and it will quite easy to forget to update this line when submitting updated versions of the I-D

Since this change is required only once, my preference is to keep the note within the YANG code as per current common practise

Reply:
2024-10-02

Agreed with comment https://github.com/ietf-ivy-wg/network-inventory-yang/pull/54#discussion_r1781561314